cmd/pebble/cmd_signal.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2021 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/pebble/client"
+)
+
+type cmdSignal struct {
+	clientMixin
+	Positional struct {
+		Signal   string   `positional-arg-name:"<SIGNAME>"`
+		Services []string `positional-arg-name:"<service>"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+var shortSignalHelp = "Send a signal to one or more running services"
+var longSignalHelp = `
+The signal command sends a signal to one or more running services. The signal
+name must be uppercase to distinguish it from the service names, for example:
+
+pebble signal HUP mysql nginx
+pebble signal SIGHUP mysql nginx    # full signal name is fine too
```

Comment:
I'm not strongly against the optional SIG prefix, but can we please wait a bit before we support it?  It feels like consistency generally wins, and from the two lines above the first one reads the best. This way we also don't even need to document the exception.

Reply:
I've removed it from the docs. However, it's still supported as we have to add the SIG prefix for the API anyway, so it would be explicit code to prevent it from working. I think consistently using `HUP` (for example) in the docs but allowing the alternative is fine.